GameServer/Logic/GameObjects/LevelProp.cs

```diff
@@ -13,8 +13,9 @@ public class LevelProp : GameObject
         private string name;
         private string type;
         private float z, dirX, dirY, dirZ, unk1, unk2;
+        private byte skin;
 
-        public LevelProp(Game game, uint id, float x, float y, float z, float dirX, float dirY, float dirZ, float unk1, float unk2, string name, string type) : base(game, id, x, y, 0)
+        public LevelProp(Game game, uint id, float x, float y, float z, float dirX, float dirY, float dirZ, float unk1, float unk2, string name, string type, byte skin = 0) : base(game, id, x, y, 0)
```

Comment:
This initializer looks horrible, but that's not something you should fix unless you feel like it.